gateway-manager/home/app/src/components/LandingPage.jsx

```diff
@@ -27,10 +27,16 @@ import { capitalize, getServices } from '../utils'
 const LandingPage = () => {
   const [username, setUsername] = useState('')
   const [tenant, setTenant] = useState('')
+  const [availableServices, setAvailableServices] = useState([])
 
   useEffect(() => {
     setUsername('') // TODO: set the actual username here
     setTenant(window.location.pathname.split('/')[1])
+    fetch(`${window.baseUrl}:8001/services`)
```

Comment:
This url only works if the kong service is public and it isn't :(

Reply:
In your Flask app, you can create and endpoint that makes the call internally for you :shrug: